packages/frontend-design-poc/src/components/PageLayout/PageLayout.tsx

```diff
@@ -1,29 +1,44 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useQuery } from 'react-query';
 import { Outlet } from 'react-router-dom';
 import { Footer, Header, Sidebar } from '..';
-import styles from './pageLayout.module.css';
+import { getIsAuthenticated } from '../../api/queries.ts';
 import { FeatureFlagKeys, useFeatureFlag } from '../../featureFlags';
+import { getStoredURL, isRedirectURL, removeStoredURL, saveURL } from '../../utils/url.ts';
+import styles from './pageLayout.module.css';
 
 export const PageLayout: React.FC = () => {
   const [companyName, setCompanyName] = React.useState<string>('Aker Solutions AS');
   const isCompany = !!companyName;
   const isTestFeatureToggleEnabled = useFeatureFlag<boolean>(FeatureFlagKeys.TestFeatureToggleEnabled);
+  const { data: isAuthenticated, isSuccess: isAuthenticatedSuccess } = useQuery('isAuthenticated', getIsAuthenticated);
+
+  console.log('isAuthenticated', isAuthenticated);
+
+  useEffect(() => {
```

Comment:
Refactor out the hook into its own file? But it in an `auth/` folder together with `urls.ts`?

Reply:
Not sure if PageLayout is the best place to put this logic, we might need to use the private router pattern. Might as well be a hook.